packages/rest/src/rest.server.ts

```diff
@@ -251,7 +242,11 @@ export class RestServer extends Context implements Server, HttpServerLike {
 
     // Allow CORS support for all endpoints so that users
     // can test with online SwaggerUI instance
-    this._expressApp.use(cors(this.config.cors));
+    this.expressMiddleware(cors, this.config.cors, {
+      injectConfiguration: false,
+      key: 'middleware.cors',
+      group: 'cors',
+    });
```

Comment:
I think we should eventually not rely on the built-in CORS middleware and modify our application template to include explicit setup of CORS.